.github/workflows/snyk.yml

```diff
@@ -10,4 +10,4 @@ jobs:
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
-          command: test
\ No newline at end of file
+          command: monitor
```

Comment:
Is this just a test PR or are you looking to go forward with this technique if it works?  If so, can you describe what this changes please?

Reply:
Test pr